### ISSUE TITLE
How to parse Domain Status directives?

### ISSUE THREAD

**Issue by postmodern** [weppos/whois#399]
It currently doesn't seem possible to parse the `Status:` or `Domain Status:` directives for a whois record:

```
Status: clientTransferProhibited http://www.icann.org/epp#clientTransferProhibited
Status: serverDeleteProhibited http://www.icann.org/epp#serverDeleteProhibited
Status: serverTransferProhibited http://www.icann.org/epp#serverTransferProhibited
Status: serverUpdateProhibited http://www.icann.org/epp#serverUpdateProhibited

Domain Status: serverTransferProhibited
Domain Status: serverDeleteProhibited
Domain Status: clientTransferProhibited
Domain Status: serverUpdateProhibited
```

**Comment by weppos** [comment]
This is a long outstanding issue #5. Every registry returns the status in a different way. Unfortunately, I discovered such issue late after the development of the first parser prototype and at the time being the `.status` method is inconsistent.

My goal is to eventually have `.status` return a `Status` object, that exposes both the list of statuses returned by the registry and the normalized domain status (registered, available, etc). The latter is always set because it's possible to guess the value from the statuses, the former is not always present (some registries don't provide status information).

Long story short, at the time being you should use `.status` but it's not guaranteed to return the list of statuses as provided by the registry.
